baseapp-profiles/baseapp_profiles/graphql/filters.py

```diff
@@ -0,0 +1,25 @@
+import django_filters
+import swapper
+from django.db.models import Q
+
+Profile = swapper.load_model("baseapp_profiles", "Profile")
+
+
+class ProfileFilter(django_filters.FilterSet):
+    q = django_filters.CharFilter(method="filter_q")
+
+    order_by = django_filters.OrderingFilter(
+        fields=(
+            ("created", "created"),
+            ("followers_count__total", "followers_count_total"),
+            ("following_count__total", "following_count_total"),
+            ("is_followed_by_me", "is_followed_by_me"),
```

Comment:
You can't just sort by `is_followed_by_me` its not a field in the model and it is not being annotated in the queryset, so lets remove it for now.